ansible/roles/ocp4-workload-migration/templates/mig-operator-subscription-downstream.yml.j2

```diff
@@ -9,5 +9,3 @@ spec:
   name: cam-operator
   source: redhat-operators
   sourceNamespace: openshift-marketplace
-  startingCSV: cam-operator.{{ mig_operator_release }}
```

Comment:
Will that introduce the possibility of future breaks? Whenever a new operator version is released?

Reply:
We have that situation now.

We were told to use "startingCSV" to avoid this and pin to a specific operator, but that isn't working.

Asking @shawn-hurley (Operator Framework group lead) for help on figuring out what else is needed to pin this down for future, perhaps we are missing something in our rel-eng workflows for the operator.

In meantime, the workload is broken until this is removed.